_playwright-tests/Integration/CanUpdateSystemWithTemplate.spec.ts

```diff
@@ -49,11 +51,12 @@ test.describe('Test System With Template', async () => {
       await page.getByText('Use up to a specific date', { exact: true }).click();
       await page.getByPlaceholder('YYYY-MM-DD', { exact: true }).fill('2021-05-17'); // Older than any snapshot date
       await page.getByRole('button', { name: 'Next', exact: true }).click();
-      await page.getByText('add template modal', { exact: true });
```

Comment:
This is interesting. The original line does not make sense. Good find!
@dominikvagner we talked that we could enable eslint rule for playwright which could catch stuff like these.